packages/components/Checkbox/src/Checkbox.settings.win32.ts

```diff
@@ -0,0 +1 @@
+export * from './Checkbox.settings.old';
```

Comment:
Ah, you do have a settings.win32... I feel like this is an unnecessary level of redirection and we could just leave a comment on top of the original file saying "These settings are win32 only because ...."